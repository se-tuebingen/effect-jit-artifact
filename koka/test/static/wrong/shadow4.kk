// combination of warning and error used to display incorrectly
fun foo(x) { val x = 2; x + True }