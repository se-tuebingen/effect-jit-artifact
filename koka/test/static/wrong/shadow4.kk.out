test/static/wrong/shadow4@kk(2,29): type error: True does not match the argument types
 context : True
 term : True
 inferred type: bool
 expected type: int
test/static/wrong/shadow4@kk(2,18): type warning: x shadows an earlier local definition or parameter